.eslintrc.yml

```diff
@@ -226,10 +226,10 @@ rules:
   # Stylistic Issues
   # https://eslint.org/docs/rules/#stylistic-issues
 
-  #array-bracket-newline: undecided
-  #array-bracket-spacing: undecided
-  #array-element-newline: undecided
-  #block-spacing: undecided
+  array-bracket-newline: off
```

Comment:
Based on this list: https://github.com/prettier/eslint-config-prettier/blob/master/index.js